### PR TITLE
correction with respect to remarks

### PR DESCRIPTION
**Pull request by herimanana301**

### CODE REVIEW

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
-import Home from "@pages/Home";
-
+import Home from "./pages/Home";
+import NavBar from "./components/navbar/Navbar";
```
Comment (rocambille) [inline frontend/src/App.jsx:2]: NavBar or Navbar : pick one :)

File: frontend/style/navbar.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    height: 45px;
+}
+header{
+    display: flex;
+    justify-content: space-between;
+}
+
+.search{
+    border-radius: 6%;
+    height: 2.4rem;
```
Comment (rocambille) [inline frontend/style/navbar.css:18]: avoid fixed height on elements : an element has the height of its content

**Review by rocambille — CHANGES_REQUESTED** [review]

File: frontend/src/App.jsx (hunk 1 above)
Comment (herimanana301) [reply frontend/src/App.jsx]: We resolved

File: frontend/style/navbar.css (hunk 2 above)
Comment (herimanana301) [reply frontend/style/navbar.css:20]: We resolved

Comment (rocambille) [reply frontend/style/navbar.css:20]: so commit and push the resolution :)

**Review by rocambille — CHANGES_REQUESTED** [review]

File: frontend/style/navbar.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
+.icon{
+    width: 6rem;
+    height: 3rem;
+    margin-top: 0.4rem;
+
```
Comment (Justine-Merlin) [inline frontend/style/navbar.css:5]: Delete blank line.

File: frontend/style/navbar.css [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+}
+.icon1{
+    width: 3rem;
+    height: 3rem;
+}
+header{
+    display: flex;
+    justify-content: space-between;
+    background-color: #24242a;
+
```
Comment (Justine-Merlin) [inline frontend/style/navbar.css:15]: Delete blank line.

**Review by Justine-Merlin — CHANGES_REQUESTED** [review]

File: frontend/src/components/navbar/Search.jsx [hunk 5]
```diff
@@ -0,0 +1,8 @@
+function Search() {
+  return (
+    <div className="NavSearch">
```
Comment (Justine-Merlin) [inline frontend/src/components/navbar/Search.jsx:3]: Correct classname would be nav-search instead of NavSearch.

File: frontend/style/navbar.css [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+.search{
+    border-radius: 6%;
+    height: 2.4rem;
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
+}
+
+.NavSearch{
```
Comment (Justine-Merlin) [inline frontend/style/navbar.css:24]: Correct classname would be nav-search instead of NavSearch.

**Pull request merged**